docs/user_manual/reports.md

```diff
@@ -0,0 +1,43 @@
+# Reports
+
+## Case QC Report
+
+This report displays a detailed view of all items for each QC step of a case, including all metrics
+and sign-off details. This report is a live Dimsum page, but is also optimized for printing,
+including to PDF as a means of saving. To view the report, click the "QC Report" button at the top
+right of the Case Details page.
+
+## TGL Tracking Sheet
+
+This report lists all of the tests for each case in a project. Progress of each test is shown by
+listing the samples or other items used for each QC step along with a few other details. To generate
+the report, go to the Projects list page, select the projects you wish to report on, and click the
+"Download" button. The report will be downloaded.
+
+## Full-Depth Summary
+
+This report lists details of all the full-depth sequencing that has been completed for the selected
+cases. The report is generated from any table of cases, including the main QC dashboard, and the
+Cases table on various detail pages. To generate the report, select the cases you are interested in,
+and click the "Download" button. In the dialog that opens, choose this report. The report will be
+downloaded.
+
+## DARE Input Sheet
+
+This report lists the library and run names of all full-depth sequencing for the selected cases. It
+is used for input into DARE. The report is generated from any table of cases, including the main QC
+dashboard, and the Cases table on various detail pages. To generate the report, select the cases you
+are interested in, and click the "Download" button. In the dialog that opens, choose this report.
+The report will be downloaded.
+
+## Case TAT Report
+
+This report lists all of the tests for each case. A detailed account of turn-around time is included
+for each test, listing the completion dates and time spent on each QC step. Some QC steps are
+further broken down into sub-steps. The report is generated from any table of cases, including the
+main QC dashboard, and the Cases table on various detail pages. To generate the report, select the
+cases you are interested in, and click the "TAT Report" button. The report will be downloaded.
+
+## TAT Trend Report
+
+(TODO)
```

Comment:
Will this TODO be addressed in this PR?

Reply:
No. This feature isn't done yet and the documentation should be written with the feature (which @wuall826  is working on)